test/functional/terminal/ex_terminal_spec.lua

```diff
@@ -79,6 +79,18 @@ describe(':terminal (with fake shell)', function()
     ]])
   end)
 
+  it('with no argument, but the shell has arguments, acts like termopen()', function()
```

Comment:
vim options are indicated with single quotes, by convention, so this can be a bit more terse:

    it("with no argument, but 'shell' has arguments, acts like termopen()", function()